packages/amplify-app/src/index.js

```diff
@@ -81,6 +81,10 @@ async function amplifyCLIVersionCheck() {
     const amplifyCLIVersion = semver.coerce(stripAnsi(amplifyCLIVersionSpawn.stdout.toString()));
     if (semver.satisfies(amplifyCLIVersion, minCLIVersion)) {
       console.log(`${emoji.get('white_check_mark')} Found Amplify CLI v${amplifyCLIVersion}`);
+    } else {
+      console.log(`${emoji.get('worried')} Amplify CLI version ${minCLIVersion} not found.`);
```

Comment:
I think a more developer friendly message would be something like:
- No Amplify CLI found: "Amplify CLI was not found"
- Non-satisfying CLI version found: "Amplify CLI version found was: ${amplifyCLIVersion}, the minimum required version is: ${minCLIVersion} 

This way we present more information about why are we installing a new CLI version.

Reply:
Updated